CMakeLists.txt

```diff
@@ -34,13 +34,13 @@ message(STATUS "PYFLANN_VERSION = ${PYFLANN_VERSION}")
 DISSECT_VERSION()
 GET_OS_INFO()
 
-if (OS_IS_MACOS)
```

Comment:
Pending resolution here for Macports vs Homebrew: https://github.com/WildbookOrg/wbia-vtool/pull/4#discussion_r444454154